### PR TITLE
Improve pipeline resilience against accidental failures

### PR DESCRIPTION
**Pull request by sergejparity**
This PR combines redundant `retry` blocks of the pipeline into `default` section

### CODE REVIEW

**Review by alvicsam — APPROVED** [review]

**Comment by HCastano** [comment]
@sergejparity @alvicsam Hey guys, I appreciate the PR, but in the future can you wait until one of the ink! team members acknowledges a CI related PR before merging?

**Pull request merged**
